.github/workflows/ci.yml

```diff
@@ -14,28 +14,28 @@ jobs:
       fail-fast: false
       matrix:
         tests:
-          - env: check
+          - env: py311-check
```

Comment:
`check` is required in the CI, but I renamed it here, should we update the CI required check to `py311-check` or keep the check and I can revert here?